apps/log-collector/Makefile

```
@@ -0,0 +1,98 @@
+# Complete development workflow: builds, deploys, and shows logs
+# Perfect for getting started or testing the full pipeline
+#
+# This target does everything in sequence:
+# 1. Builds Docker image (if Dockerfile/src/fluent-bit changed)
+# 2. Creates namespace
+# 3. Generates ConfigMap from .env.local
+# 4. Applies all Kubernetes resources
+# 5. Restarts deployments to ensure latest ConfigMap is used
+# 6. Waits for pods to be ready
+.PHONY: dev
+dev: $(ROOT_DIR)/apps/log-collector/Dockerfile $(ROOT_DIR)/apps/log-collector/src $(ROOT_DIR)/apps/log-collector/fluent-bit $(ROOT_DIR)/apps/log-collector/k8s
```

checkmake (0.2.2)

[warning] Target body for "dev" exceeds allowed length of 5 (19).

(maxbodylength)